manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -1985,6 +1985,7 @@ def configure_active_directory(self, security_service, vserver_name):
                 LOG.debug(f"Attempt ({attempt}): Trying to setup CIFS server "
                           f"with args: {api_args}")
                 self.send_request('cifs-server-create', api_args)
+                self.configure_cifs_signing()
```

Comment:
we might need to wait for cifs-server-create, especially in production?

Reply:
Yes, that could be a problem - I added to wait for a cifs server first